src/ert/ensemble_evaluator/_ensemble.py

```diff
@@ -278,13 +272,18 @@ async def _evaluate_inner(  # pylint: disable=too-many-branches
                 ),
                 exc_info=True,
             )
-            await cloudevent_unary_send(event_creator(EVTYPE_ENSEMBLE_FAILED, None))
+            await event_unary_send(event_creator(EVTYPE_ENSEMBLE_FAILED))
             return
 
         logger.info(f"Experiment ran on QUEUESYSTEM: {self._queue_config.queue_system}")
 
         # Dispatch final result from evaluator - FAILED, CANCEL or STOPPED
-        await cloudevent_unary_send(event_creator(result, None))
+        await event_unary_send(event_creator(result))
+
+    async def update_scheduler_running(self) -> None:
+        assert self._scheduler is not None
+        await self._scheduler.running.wait()
+        self.scheduler_started.set()
```

Comment:
Why is this `scheduler_running` needed in the first place?

Reply:
Changed it so that cancel_all_jobs in scheduler now will wait until tasks have started before trying to cancel them. Removed the scheduler_running parts.